modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCommerceDestinationNamesCheck.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringUtil;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class UpgradeJavaCommerceDestinationNamesCheck
```

Comment:
@michaelmdelcavalcanti, did you look for on Master if hasn't cases on JSP? If so, override the extensions method, rename the check to remove the "java" from its name, and create the JSP test files.

Reply:
There are no cases in the JSP files.